.github/actions/build-khiops/action.yml

```diff
@@ -29,9 +29,16 @@ runs:
         sudo apt-get install openmpi-bin libopenmpi-dev
         sudo update-alternatives --set mpi /usr/bin/mpicc.openmpi
         sudo update-alternatives --set mpirun /usr/bin/mpirun.openmpi
-    - name: Setup Ninja
-      uses: ashutoshvarma/setup-ninja@v1.1
-    - name: Load Visual C++ Environment Variables (Windows only)
+    - name: Setup Ninja (Linux)
+      if: runner.os == 'Linux'
+      shell: bash
+      run: sudo apt-get install ninja-build
+    - name: Setup Ninja (macOS)
```

Comment:
We do not build native packages on Mac OS, do we? And for Conda, this is managed by Conda itself, right? So, I would remove the MacOS part for now.

Reply:
Ok to stay isofunctional to what we had before.